domain/build.gradle.kts

```diff
@@ -1,7 +1,25 @@
+import org.springframework.boot.gradle.tasks.bundling.BootJar
+
 plugins {
+    id("org.springframework.boot") version "3.2.5"
+    id("io.spring.dependency-management") version "1.1.4"
 }
 
 version = "0.0.1-SNAPSHOT"
 
 dependencies {
+    implementation("org.springframework:spring-context:6.1.6")
+
+
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
+    implementation("org.springframework.boot:spring-boot-starter-web")
```

Comment:
When building hexagonal or ports/adapters, keep your domain dependencies clean. Most of the time spring-boot-starter dependencies come with way more than you actually need.

Investigate which part of the spring-boot-starter-web dependency you need and maybe add the specific part as a depencency?